ParisBNF/abb/BNFabb119.xml

```diff
@@ -319,12 +322,12 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </msItem>
                                 </msItem>
```

Comment:
delete /msItem here, move it before ms_i4

Reply:
I think I did it.